src/scripts/core/services/score-maths.ts

```diff
@@ -0,0 +1,78 @@
+import { Token } from "../models/token";
+
+export class ScoreMaths {
+
+    static scoreMap: Map <number, number> = new Map <number, number>([
+        [1, 5],
+        [2, 10],
+        [3, 25],
+        [4, 50],
+        [5, 100],
+        [6, 200],
+        [7, 250],
+        [8, 500],
+        [9, 1000]
+    ]);
+
+    //static tokens: Token[] = [];
+
+    constructor() {
```

Comment:
If you have an empty constructor (Or one that only calls super with the same parameters as the constructor), you don't actually need to declare it here